models/repo.go

```diff
@@ -1451,7 +1451,14 @@ func GetRepositoryByID(id int64) (*Repository, error) {
 
 // GetUserRepositories returns a list of repositories of given user.
 func GetUserRepositories(userID int64, private bool, page, pageSize int) ([]*Repository, error) {
-	sess := x.Where("owner_id = ?", userID).Desc("updated_unix")
+	sess := x.Where("owner_id = ?", userID).Desc("uid").Desc("updated_unix")
+
+	if setting.UsePostgreSQL {
```

Comment:
Just look at the XORM docs, or maybe @lunny can give you a hint since he's the XORM author ;)